src/apps/Console/ConsoleFeedback.js

```diff
@@ -171,7 +174,7 @@ export default function ConsoleFeedback({
               `}
               onClick={() => handleClick(app.proxyAddress)}
             >
-              {`${app.name} (${app.proxyAddress})`}
+              {`${app.name} (${shortenAddress(app.proxyAddress)})`}
```

Comment:
Should we keep the full address as a title?

```suggestion
              <span>{app.name}</span> (<span title={app.proxyAddress}>{shortenAddress(app.proxyAddress)}</span>)
```

Reply:
Aah yes, missed this